src/FluentMigrator.DotNet.Cli/Commands/ConnectionCommand.cs

```diff
@@ -25,17 +25,20 @@ namespace FluentMigrator.DotNet.Cli.Commands
     public class ConnectionCommand : MigrationCommand
     {
         [Option("-c|--connection <CONNECTION_STRING_OR_NAME>", Description = "The name of the connection string (falls back to machine name) or the connection string itself to the server and database you want to execute your migrations against.")]
-        public string ConnectionString { get; }
+        public string ConnectionString { get; set; }
+
+        [Option("-m|--transaction-mode <MODE>", Description = "Overrides the transaction behavior of migrations, so that all migrations to be executed will run in one transaction.")]
+        public TransactionMode TransactionMode { get; set; }
 
         [Option("--no-connection", Description = "Indicates that migrations will be generated without consulting a target database. Should only be used when generating an output file.")]
         public bool NoConnection { get; }
 
         [Option("-p|--processor <PROCESSOR_NAME>", Description = "The kind of database you are migrating against. Available choices can be shown with \"list processors\".")]
         [Required]
-        public string ProcessorType { get; }
+        public string ProcessorType { get; set; }
```

Comment:
Remove set;

Reply:
In the official examples only boolean properties has only `getter`.

https://github.com/natemcmaster/CommandLineUtils/blob/master/docs/samples/subcommands/inheritance/Program.cs

Reply:
You can see in the [README.md](https://github.com/natemcmaster/CommandLineUtils/blob/1fefc7348f1007d32f5378f180b1cd45926664c6/README.md) that all command line argument properties are missing their setters.

Reply:
Should I follow the README or the examples? I'm confused and I think even they are.

Reply:
I think the README is the right way. The bottom line is it doesnt make sense to expose a setter unless you need to for testing purposes.  Even so, you probably are testing things incorrectly if you need a setter to configure your objects (even though I love doing this).